extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -156,7 +156,7 @@ export class AmpAdApiHandler {
   updateSize_(height, width) {
     const targetOrigin =
           this.iframe_.src ? parseUrl(this.iframe_.src).origin : '*';
-    this.baseInstance_.attemptChangeSize(height, width, () => {
+    this.baseInstance_.attemptChangeSize(height, width).then(() => {
```

Comment:
We have tests for `updateSize_` but we only test that `attemptChangeSize(height, width)` are been called. 
You are right  we need to test `#postMessage`  is called. 
#4256